packages/nextjs/app/blockexplorer/_components/TransactionHash.tsx

```diff
@@ -2,12 +2,20 @@
 
 import { useState } from "react";
 import Link from "next/link";
-import CopyToClipboard from "react-copy-to-clipboard";
 import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
 
 export const TransactionHash = ({ hash }: { hash: string }) => {
   const [addressCopied, setAddressCopied] = useState(false);
 
+  const handleCopy = () => {
+    navigator.clipboard.writeText(hash).then(() => {
+      setAddressCopied(true);
+      setTimeout(() => {
+        setAddressCopied(false);
+      }, 800);
+    });
+  };
```

Comment:
It would be nice to use async / await instead of .then

Reply:
Implemented in new hook.